client/src/components/components_pro-inscription/DetailDescriptionComponent.jsx

```diff
@@ -0,0 +1,296 @@
+          <h2 className="titles">Les petits plus de votre accueil</h2>
+          <p className="texts">
+            {" "}
+            Il s'agit en général des services que les parents souhaitent
+            retrouver pour l'accueil de leurs enfants. Vous pourrez ajouter
+            d'autres après la publication.
+          </p>
+          <section className="detail-checks">
+            <h3 className="titles little-titles-details">Formations</h3>
+            <section
+              className="checkboxgroup-detail"
+              aria-label="Type de formations"
+            >
+              <section>
+                <input
+                  id="pcsc1"
+                  name="type"
+                  type="checkbox"
+                  className="input-checkbox-detail-pro"
+                />
+                <label htmlFor="pcsc1" className=" texts">
+                  Formation premier secours (PCSC1)
+                </label>
+              </section>
+              <section>
+                <input
+                  id="nesting"
+                  name="type"
+                  type="checkbox"
+                  className="input-checkbox-detail-pro"
+                />
+                <label htmlFor="nesting" className=" texts">
+                  Formation Nesting (pollution intérieure)
+                </label>
+              </section>
+              <section>
+                <input
+                  id="montessori"
+                  name="type"
+                  type="checkbox"
+                  className="input-checkbox-detail-pro"
+                />
+                <label htmlFor="montessori" className=" texts">
+                  Pédagogie Montessori / Pikler Loczy
+                </label>
+              </section>
+              <section>
+                <input
+                  id="disability"
+                  name="type"
+                  type="checkbox"
+                  className="input-checkbox-detail-pro"
+                />
+                <label htmlFor="disability" className=" texts">
+                  Formation accueil d'enfant handicapés
+                </label>
+              </section>
+            </section>
+          </section>
+          <section className="detail-checks">
+            <h3 className="titles little-titles-details">Accueil</h3>
+            <section
+              className="checkboxgroup-detail"
+              aria-label="Type d'acccueil"
+            >
+              <section>
+                <input
+                  id="garden"
+                  name="type"
+                  type="checkbox"
+                  className="input-checkbox-detail-pro"
+                />
+                <label htmlFor="garden" className=" texts">
+                  Espace extérieur / jardin
+                </label>
+              </section>
+              <section>
+                <input
+                  id="outing"
+                  name="type"
```

Comment:
les inputs portent tous le même nom ? ne devrait ils pas porter le nom du label htmlFor ?

Reply:
C'est exact. Exemple ci-dessous :

`<label for="name">Name (4 to 8 characters):</label>
<input type="text" id="name" name="name" required minlength="4" maxlength="8" size="10" />`

C'est redondant, mais c'est la norme.